contracts/mocks/Base64Mock.sol

```diff
@@ -0,0 +1,11 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import "../utils/Base64.sol";
+
+contract Base64Mock {
```

Comment:
just my 2 cents but as an alternative, you can have the following line at the top of the contract:
`using Base64 for bytes;`

and then in your encode:
`value.encode();`